test/functional/unified-spec-runner/entities.ts

```diff
@@ -290,7 +391,7 @@ export class EntitiesMap<E = Entity> extends Map<string, E> {
         }
 
         map.set(entity.bucket.id, new GridFSBucket(db, options));
-      } else if ('stream' in entity) {
+      } else if ('cursor' in entity) {
         throw new Error(`Unsupported Entity ${JSON.stringify(entity)}`);
```

Comment:
It is supported right? It just needs to be created via an operation, so maybe change the error message here

Reply:
That was an error in my part, I've reverted the change.

Reply:
I reverted. It is supported but only via a operation.